configs/manager/halfstack.toml

```diff
@@ -12,6 +12,7 @@ name = "backend"
 user = "postgres"
 password = "develove"
 pool-recycle = 50
+pool-pre-ping = true
```

Comment:
Since the default value is false, I think this should be also false.
Though, we should recommend setting this option for HA setups.